Bug: Fix `make $(TMPGOPATH)/bin/gotext` target

`make $(TMPGOPATH)/bin/gotext` target fails because `go/types` in go 1.21.x doesn't have 1 of the functions:

```
make /tmp/anax-gopath/bin/gotext
Fetching gotext
# golang.org/x/text/message/pipeline
/tmp/anax-gopath/pkg/mod/golang.org/x/text@v0.22.0/message/pipeline/extract.go:151:21: undefined: types.Unalias
/tmp/anax-gopath/pkg/mod/golang.org/x/text@v0.22.0/message/pipeline/extract.go:157:23: undefined: types.Unalias
```

--- a/Makefile
+++ b/Makefile
@@ -604,7 +604,7 @@ $(TMPGOPATH)/bin/gotext:
 	if [ ! -e "$(TMPGOPATH)/bin/gotext" ]; then \
 		echo "Fetching gotext"; \
 		export GOPATH=$(TMPGOPATH); export PATH=$(TMPGOPATH)/bin:$$PATH; \
-			go install golang.org/x/text/cmd/gotext@latest; \
+			go install golang.org/x/text/cmd/gotext@v0.21.0; \
 	fi
 
 gopathlinks:

